fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/provider/FabricLanguageProvider.java

```diff
@@ -228,6 +229,16 @@ default void add(Identifier identifier, String value) {
 			add(identifier.toTranslationKey(), value);
 		}
 
+		/**
+		 * Adds a translation for an {@link TagKey}.
+		 *
+		 * @param tagKey The {@link TagKey} to get the translation key from.
+		 * @param value      The value of the entry.
```

Comment:
```suggestion
		 * @param tagKey The {@link TagKey} to get the translation key from.
		 * @param value  The value of the entry.
```
Nit: the formatting is a bit odd here